docs/components/organization/organization-list.mdx

```diff
@@ -0,0 +1,169 @@
+---
+
+# `<OrganizationList />`
+
+<Images
+  alt="Organization List Example"
+  width={992}
+  height={568}
+  src="/docs/images/ui-components/component-org_list.png"
+/>
+
+The `<OrganizationList />` component is used to display organization related memberships, invitations, and suggestions for the user.
+
+## Usage
+
+<Tabs items={['Next.js', 'React', 'Remix']}>
+  <Tab>
+    <CodeBlockTabs options={["App Router", "Pages Router"]}>
+        ```jsx copy filename="/app/discover/page.[jsx/tsx]"
+      import { OrganizationList } from "@clerk/nextjs";
+
+      export default function OrganizationListPage() {
+        return (
+            <OrganizationList
+                afterCreateOrganizationUrl='/organization/:slug'
+                afterSelectPersonalUrl='/user/:id'
+                afterSelectOrganizationUrl='/organization/:slug'
+            />
+        );
+      }
+      ```
+
+        ```jsx copy filename="/pages/discover.[jsx/tsx]"
+      import { OrganizationList } from "@clerk/nextjs";
+
+      export default function OrganizationListPage() {
+        return (
+            <OrganizationList
+                afterCreateOrganizationUrl={org => `/organization/${org.slug}`}
+                afterSelectPersonalUrl={user => `/user/${user.id}`}
+                afterSelectOrganizationUrl={org => `/organization/${org.slug}`}
+            />
+        );
+      }
+      ```
+    </CodeBlockTabs>
+  </Tab>
+
+  <Tab>
+    ```jsx copy filename="discover.tsx"
+    import { OrganizationList } from "@clerk/clerk-react";
+
+    export default function OrganizationListPage() {
+      return (
+          <OrganizationList
+              afterCreateOrganizationUrl={org => `/organization/${org.slug}`}
+              afterSelectPersonalUrl={user => `/user/${user.id}`}
+              afterSelectOrganizationUrl={org => `/organization/${org.slug}`}
+          />
+      );
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```jsx copy filename="/route/discover.tsx"
+    import { OrganizationList } from "@clerk/remix";
+
+    export default function OrganizationListPage() {
+      return (
+          <OrganizationList
+              afterCreateOrganizationUrl={org => `/organization/${org.slug}`}
+              afterSelectPersonalUrl={user => `/user/${user.id}`}
+              afterSelectOrganizationUrl={org => `/organization/${org.slug}`}
+          />
+      );
+    }
+    ```
+  </Tab>
+</Tabs>
+
+### Force Organizations
+
+For specific applications, we want to prohibit people from using their personal accounts as they need to be part of an organization.
+Use <code>{"<OrganizationList/>"}</code> with <code>{"hidePeronsal"}</code> in order to guide your user to join or create an organization in order to continue.
+```tsx copy filename="organization-portal.tsx"
+export default function OrganizationListPage() {
+  return (
+    <OrganizationList
+        hidePersonal={true}
+        {...}
+    />
+  );
+};
+```
+
+## Props
+
+All props below are optional.
+
+<Tables
```

Comment:
This table does not require complex use of JSX components, so let's use a markdown table.

| Name | Type | Description |
| --- | --- | --- |
| `hidePersonal` | `boolean` | By default, users can switch between organization and their personal account. This option controls whether `OrganizationList` will include the user's personal account in the organization list. Setting this to `false` will hide the personal account entry, and users will only be able to switch between organizations.<br />Defaults to `false`.

etc etc etc